lib/identity/finalize.rb

```diff
@@ -0,0 +1,30 @@
+module Identity
+  class Finalize < Default
```

Comment:
I know that the existing convention is a bit light, but can you add an explanatory comment indicating what this module is for?